src/components/Signup/PodRegistrationForm.jsx

```diff
@@ -0,0 +1,152 @@
+// React Imports
+import React, { useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+// MUI imports
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+import { Visibility, VisibilityOff } from '@mui/icons-material';
+import FilledInput from '@mui/material/FilledInput';
+import IconButton from '@mui/material/IconButton';
+import InputAdornment from '@mui/material/InputAdornment';
+import InputLabel from '@mui/material/InputLabel';
+import Button from '@mui/material/Button';
+import Card from '@mui/material/Card';
+import CardHeader from '@mui/material/CardHeader';
+
+/* Styles for MUI components */
```

Comment:
These can probably be done in line with the MUI components. We don't really have separate styling used elsewhere. Right now much of it is the default. Margin can be included using `sx={{ margin: 8 }}`, for example.

Reply:
Got it,  I'll make those inline.